x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/field_parameters/inference_id_selects.tsx

```diff
@@ -79,19 +61,26 @@ export const InferenceIdSelects = ({ onChange, 'data-test-subj': dataTestSubj }:
             )}
           </UseField>
         </EuiFlexItem>
-        <EuiFlexItem grow={true}>
-          <EuiCallOut
-            size="s"
-            color="warning"
-            title={i18n.translate(
-              'xpack.idxMgmt.mappingsEditor.parameters.noReferenceModelStartWarningMessage',
-              {
-                defaultMessage:
-                  'The referenced model for this inference endpoint will be started when adding this field.',
-              }
-            )}
-          />
-        </EuiFlexItem>
+        {showWarning && (
+          <EuiFlexGroup gutterSize="s" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <EuiIcon color="warning" type="warning" />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiCallOut
+                size="s"
+                color="warning"
+                title={i18n.translate(
+                  'xpack.idxMgmt.mappingsEditor.parameters.noReferenceModelStartWarningMessage',
+                  {
+                    defaultMessage:
+                      'The referenced model for this inference endpoint will be started when adding this field.',
```

Comment:
Quick question. Will the status be hidden when referenced model is third party integration inference id? 
If I understand correctly this callout would be still there if there is an error

Reply:
The status should be hidden when the referenced model comes from a third-party integration inference ID. As far as I understand, we should display the message if the underlying trained models (downloaded on ML nodes) have not been deployed. I'll review the code to confirm whether this behavior is implemented in the current change.

Reply:
This [comment](https://github.com/elastic/kibana/pull/180246/commits/09df652d803523756bb50746a75d61737a980bfd) should address this issue